src/lib/time-picker/time-picker-foundation.ts

```diff
@@ -654,6 +656,12 @@ export class TimePickerFoundation implements ITimePickerFoundation {
       leadingOptions.forEach((o, index) => times.splice(index, 0, o));
     }
 
+    // Check if we need to show the Hours options
+    if (!this._showHourOptions) {
+      // empty times array
+      times.splice(0, times.length);
+    }
```

Comment:
I still think rather than clearing it out, you could just wrap [time-picker-foundation.ts#L621-L636](https://github.com/tyler-technologies-oss/forge/blob/41b2fc1b76aee1d21a32173e61ebcaab13652298/src/lib/time-picker/time-picker-foundation.ts#L621-L636) in `if (this._showHourOptions)` and then let Now and the custom options just get added to the empty array.  Just seems better to not do the work, then do it and then undo it.

Might also need to exclude the divider on line 655 if `times.length === 0`.

Reply:
Both fixed with most recent push